test/acceptance/features/environment.py

```diff
@@ -14,12 +14,18 @@
 
 from steps.command import Command
 from steps.environment import ctx
+from behave.model_core import Status
 
 import os
 import semver
 
 cmd = Command()
 
+if os.getenv("DELETE_NAMESPACE") in ["alwaysdelete", "neverdelete", "keepwhenfailed"]:
```

Comment:
Nit: I like this better:
```suggestion
if os.getenv("DELETE_NAMESPACE") in ["always", "never", "keepwhenfailed"]:
```
This would also require lines 74 and 76 to be changed.